src/WebJobs.Script/Content/Script/functions.js

```diff
@@ -49,7 +51,15 @@ function createFunction(f) {
         });
         context.log = log;
 
+        if (typeof func === "string") {
```

Comment:
We used to pass in a function.  I maintained this behavior because it makes testing far easier (don't need separate function files to require).  In the prod code it's all string, however